addons/functions/functions/fnc_applyFatigue.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/*
+ * Author:  nomisum
+ * Apply Fatigue on Client
+ *
+ * Arguments:
+ * 0: Trench <OBJECT>
+ * 1: Unit <OBJECT>
+ *
+ * Return Value:
+ * NONE
+ *
+ * Example:
+ * [trenchObj, unitObj] call grad_trenches_functions_fnc_applyFatigue
+ *
+ * Public: No
+ */
+
+// called every 1s on builder
+
+params ["_trench", "_unit"];
+
```

Comment:
Ther should be a check here if the unit is local. Just to be sure.

Reply:
this is called by a target event – it cant be not-local. 

the worst that can happen is the server fatigue gets drawn out as this is called from server. not sure if the server has stamina though :D

Reply:
Hm, just thinking about how to make it PEBKAC proof

Reply:
if that is the goal we need null checks for each and everything too. not sure if this is the quality level we aim for 😁 would bloat up the project quite a bit.

Reply:
Thats true.